front/pages/api/w/[wId]/assistant/conversations/[cid]/messages/index.ts

```diff
@@ -11,6 +11,16 @@ import { apiError, withLogging } from "@app/logger/withlogging";
 
 const PostMessagesRequestBodySchema = t.type({
   content: t.string,
+  mentions: t.array(
+    t.union([
+      t.type({ id: t.number, configurationId: t.string }),
```

Comment:
no ModelId here

We might want to change the type of the function signature or maybe remove the id:ModelId from MentionType if we don't use it anywhere (I think we don't)